src/integrationTest/java/uk/gov/hmcts/reform/laubackend/cases/bdd/SpringBootIntegrationTest.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.laubackend.cases.bdd;
+
+import io.cucumber.spring.CucumberContextConfiguration;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.laubackend.cases.Application;
+
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+
+
+@CucumberContextConfiguration
+@SpringBootTest(classes = Application.class, webEnvironment = RANDOM_PORT)
+@Slf4j
+@SuppressWarnings({"PMD.AbstractClassWithoutAnyMethod", "PMD.AbstractClassWithoutAbstractMethod"})
+public abstract class SpringBootIntegrationTest {
+
+    @BeforeAll
+    public void setUp() {
+        log.info("-------------- Spring Context Initialized For Executing Cucumber Tests --------------");
+    }
+}
```

Comment:
@test methods are expected in the test class. These test methods are expected to invoke the required step functions